odd-platform-specification/openapi.yaml

```diff
@@ -3883,4 +3884,25 @@ paths:
               schema:
                 $ref: './components.yaml/#/components/schemas/DataEntityGraphRelationshipDetails'
       tags:
-        - relationship
\ No newline at end of file
+        - relationship
+
+  /api/genai/question:
```

Comment:
you can ignore it
**nit**: question is quite strict word. How about `ask` there is no guarantee that it's gonna be just questioning. It's also requests like "give me example of `something` configuration example"
```suggestion
  /api/genai/ask:
```

Reply:
done